src/funcTest/groovy/info/solidsoft/gradle/pitest/functional/Junit5FunctionalSpec.groovy

```diff
@@ -11,8 +11,10 @@ class Junit5FunctionalSpec extends AbstractPitestFunctionalSpec {
         given:
             copyResources("testProjects/junit5kotlin", "")
         when:
-            ExecutionResult result = runTasksSuccessfully('pitest')
+            ExecutionResult result = runTasks('pitest')
         then:
+            !result.standardError.contains("Build failed with an exception")
```

Comment:
Wouldn't `!result.failure` be enough?

Reply:
`!result.failure` does not give any clue what went wrong. I left it as a last-resort check, in case a build fails but does not print anything to the standard error.

Reply:
You are right. Diff would not be displayed.